types.go

```diff
@@ -666,6 +666,13 @@ type Container struct {
 	Tags    string
 }
 
+type ContainerInterface struct {
+	Hwaddr string `json:"hwaddr,omitempty`
```

Comment:
I think we should rename this to `HardwareAddress` or by go "standards" it should be `HWAddr` since it's an acronym. Your choice which you prefer.

Reply:
yeah agree.